Client/Frontend/Browser/Browser.swift

```diff
@@ -45,7 +45,9 @@ class Browser: NSObject, WKScriptMessageHandler {
 
     var title: String? {
         if let title = webView.title {
-        	return title
+            if !title.isEmpty {
+                return title
```

Comment:
I wouldn't say it's unrelated (although I did split it into a separate commit) -- it fixes the tab to show "about:home" instead of an empty title in the tab tray. Both desktop and Android show the URL if the title is empty ([test page](https://people.mozilla.org/~bnicholson/test/empty-title.html)). But yeah, `displayTitle` is probably a good idea so we don't mix them up.